src/main.c

```diff
@@ -157,12 +157,14 @@ static void cardano_main(void)
 				if (e >= _ERR_AUTORESPOND_START && e < _ERR_AUTORESPOND_END) {
 					io_send_buf(e, NULL, 0);
 					flags = IO_ASYNCH_REPLY;
-#ifdef HAVE_NBGL
+					#ifdef HAVE_NBGL
 					if (e != ERR_REJECTED_BY_USER) {
-					    ui_idle();
-					    display_error();
+						ui_idle();
+						display_error();
 					}
-#endif
+					#else
+					ui_idle();
```

Comment:
I'm not sure why this was left out in #32 , but it breaks most of our tests (those testing rejection of invalid data), so I put it back.